### DISCUSSION TITLE
Welcome to corkscrew Discussions!

### DISCUSSION THREAD

**Discussion by patpadgett** [patpadgett/corkscrew#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by polarbub** [comment]
I have a proxy set up with apache's mod proxy but it doesn't want to connect to localhost. Here is the apache config https://pastebin.com/RXhz2X3G . Here is the .ssh/config `ProxyCommand /usr/bin/corkscrew localhost 8080 %h %p`. Here's the error 
```
$ ssh localhost 
Proxy could not open connnection to localhost:  Proxy Error
kex_exchange_identification: Connection closed by remote host
Connection closed by UNKNOWN port 65535
```

I put this here because the issues page wasn't open. Any ideas?

**Reply by polarbub** [reply]
Well this is the wrong spot, I didn't realize that I could create new discussions
